vyper/parser/stmt.py

```diff
@@ -534,7 +534,7 @@ def parse_for(self):
                 "..` or `for i in range(start, start + rounds): ..`"
             ), self.stmt.iter)
 
-        block_scope_id = id(self.stmt.orelse)
```

Comment:
i guess the original was a little weird

Reply:
Is there any concern here?

Reply:
@charles-cooper BTW with the new AST classes we have a node_id attribute now :) So we can replace all these hacks with using .node_id. Nice when old hacks are not necessary anymore.